### PR TITLE
feat(dtl): fast goerli sync

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Experimental release

### CODE REVIEW

**Comment by tynes** [comment]
This change isn't particularly testable without syncing, can you confirm that this syncs without any issues?

**Comment by smartcontracts** [comment]
@tynes Circle has synced a Goerli node using this change

**Review by mslipper — APPROVED** [review]

**Review by zhwrd — APPROVED** [review]

**Comment by smartcontracts** [comment]
Needs review from @tynes

File: packages/data-transport-layer/src/services/l1-ingestion/service.ts [hunk 1]
```diff
@@ -410,6 +410,33 @@ export class L1IngestionService extends BaseService<L1IngestionServiceOptions> {
         fromBlock: l1BlockRangeStart,
         toBlock: toL1Block,
       })
+    } else if (this.options.l2ChainId === 420) {
+      if (l1BlockRangeStart < 7260849) {
```
Comment (zhwrd) [inline packages/data-transport-layer/src/services/l1-ingestion/service.ts:414]: Getting an error when syncing past this block:

```
{"level":30,"time":1670021123131,"highestSyncedL1Block":7259320,"targetL1Block":7264320,"msg":"Synchronizing events from Layer 1 (Ethereum)"}
{"level":40,"time":1670021123919,"message":"TransactionEnqueued: missing event: TransactionEnqueued","msg":"recovering from a missing event"}
```

**Review by tynes — CHANGES_REQUESTED** [review]
Needs a confirmation that this actually works before its merged

**Review by zhwrd — APPROVED** [review]
confirmed the changes work

**Pull request merged**